### PR TITLE
asset-swapper: log RFQ maker (un)blacklistings

### PR DESCRIPTION
**Pull request by feuGeneA**
## Description

Follow-up to https://github.com/0xProject/0x-monorepo/pull/2706
For re-use in new module `src/utils/rfq_maker_blacklist`:
-   promoted type `LogFunction` to module `src/types', and
-   promoted default logger function declarations to module `src/constants`.

## Testing instructions

Run the `rfq_maker_blacklist` unit tests and observe the logging output on the console.

## Types of changes

* New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/asset-swapper/src/utils/rfq_maker_blacklist.ts [hunk 1]
```diff
@@ -18,6 +23,10 @@ export class RfqMakerBlacklist {
             if (this._makerTimeoutStreakLength[makerUrl] === this._timeoutStreakThreshold) {
                 this._makerBlacklistedUntilDate[makerUrl] =
                     Date.now() + this._blacklistDurationMinutes * constants.ONE_MINUTE_MS;
+                this.infoLogger(
```
Comment (steveklebanoff) [inline packages/asset-swapper/src/utils/rfq_maker_blacklist.ts:26]: super nit but could we assign `Date.now() + this._blacklistDurationMinutes * constants.ONE_MINUTE_MS;` to a variable and then use it on line 25 and this line ?

**Review by steveklebanoff — APPROVED** [review]
one nit but LGTM

File: packages/asset-swapper/src/utils/rfq_maker_blacklist.ts (hunk 1 above)
Comment (feuGeneA) [reply packages/asset-swapper/src/utils/rfq_maker_blacklist.ts:26]: Addressed in https://github.com/0xProject/0x-monorepo/pull/2714/commits/84862f15c1cb6551748102a9a101b0a04dedc8e1

**Comment by feuGeneA** [comment]
Verified on 0x-api staging:
image

**Comment by feuGeneA** [comment]
and just to round it out:
image

**Review by steveklebanoff — APPROVED** [review]

**Pull request merged**